endpoints_management/config/service_config.py

```diff
@@ -118,6 +97,56 @@ def _get_env_var_or_raise(env_variable_name):
     return os.environ[env_variable_name]
 
 
+def _make_service_config_request(service_name, service_version='', **kwargs):
```

Comment:
Why does this function take optional kwargs? There doesn't seem to be any real use for them.

Reply:
It was there as a way of passing values to request() as I was thinking there would be more uses of this. But as it turns out for now it's not used so we can remove it.